cpp/src/io/parquet/experimental/hybrid_scan_chunking.cu

```diff
@@ -141,7 +140,7 @@ void hybrid_scan_reader_impl::setup_next_pass(std::vector<rmm::device_buffer> co
     //   confuses the chunked reader
     detect_malformed_pages(pass.pages,
                            pass.chunks,
-                           (options.get_num_rows().has_value() or options.get_skip_rows() > 0)
+                           uses_custom_row_bounds(read_mode::READ_ALL)
```

Comment:
`uses_custom_row_bounds(read_mode::READ_ALL)` returns the same thing so no change here